user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md

```diff
@@ -4,52 +4,62 @@ uid: Supported_system_data_storage_architectures
 
-## Cassandra Cluster setup with Elasticsearch
+## Cassandra Cluster or Amazon Keyspaces Service combined with Elasticsearch, OpenSearch, or Amazon OpenSearch Service
 
-The **recommended** DataMiner setup involves **one Cassandra cluster and one Elasticsearch cluster for the entire DataMiner System** (DMS). This allows you to scale the database for the entire DMS at once, unlike setups with a Cassandra cluster per individual DataMiner Agent (DMA).
+The **recommended** DataMiner setup can be either on premises or in the cloud, or a mix of both. You will need:
+
+- One **Cassandra cluster** for the entire DataMiner System (DMS) or the [Amazon Keyspaces Service on AWS](xref:Amazon_Keyspaces_Service). Unlike setups with a Cassandra cluster per individual DataMiner Agent (DMA), this allows the database to be scaled for the entire DMS at once.
+
+- One **Elasticsearch** or **OpenSearch** cluster for the entire DMS or the [Amazon OpenSearch Service on AWS](xref:Amazon_OpenSearch_Service).
 
 > [!TIP]
-> For information on how to implement this setup based on an existing DataMiner setup with SQL or Cassandra databases per DMA, see [Migrating the general database to a DMS Cassandra cluster](xref:Migrating_the_general_database_to_a_DMS_Cassandra_cluster)
+> For information on how to implement a setup like this based on an existing DataMiner setup with SQL or Cassandra databases per DMA, see [Migrating the general database to a DMS Cassandra cluster](xref:Migrating_the_general_database_to_a_DMS_Cassandra_cluster)
 
-> [!IMPORTANT]
-> From DataMiner 10.3.0/10.3.3 onwards, instead of a Cassandra cluster, you can use the [Amazon Keyspaces Service on AWS](xref:Amazon_Keyspaces_Service), and instead of an on-prem Elasticsearch cluster, you can use an on-prem [OpenSearch cluster](xref:OpenSearch_database) or the [Amazon OpenSearch Service on AWS](xref:Amazon_OpenSearch_Service) (which offers both Elasticsearch and OpenSearch).
+> [!NOTE]
+> Using managed services from a cloud provider instead of on-premises databases is supported from DataMiner 10.3.0/10.3.3 onwards. Using an on-premises OpenSearch database is also supported from DataMiner 10.3.0/10.3.3 onwards.
 
 ### Single DMA setups
 
-In a development environment with limited load, it is possible to host DataMiner, Cassandra, and Elasticsearch on one Windows machine. In this case, Elasticsearch and DataMiner must be installed on a separate disk or partition. However, note that this is not recommended for normal production environments.
+We recommend running DataMiner, Cassandra, and Elasticsearch/OpenSearch on **dedicated machines**, or using **managed services from a cloud provider**. At present, we support the Amazon Keyspaces Service and Amazon OpenSearch Service. DataMiner currently still needs to run on premises, but we intend to soon make it possible to deploy a DataMiner node as a service.
```

Comment:
_DataMiner currently still needs to run on premises_

Maybe reword this to _still needs to be managed_?
Technically you can host your dma not on prem